settings.go

```diff
@@ -15,6 +15,8 @@ const (
 	DefaultStatsdPort = 8125
 	// DefaultFlushIntervalS is the default flushing interval in seconds.
 	DefaultFlushIntervalS = 5
+	// DefaultLoggingSinkEnabled is the default behavior of logging sink enablement, default is false.
+	DefaultLoggingSinkEnabled = false
```

Comment:
```suggestion
	DefaultLoggingSinkDisabled = false
```
Let's not change the default behavior.